science.tex

```diff
@@ -1,19 +1,28 @@
 \section{Science Drivers} \label{sec:science}
 
-The various different science drivers outlined in \ref{sec:science} naturally lead to different priorities for template generations, e.g. solar system science prefers templates to be generated in the NES and Milky Way science would to would prefer templates for the galactic plane to optimise alert production in these areas in early operations. Other science will prefer templates in a number of filters to enable .. rather that larger area. 
+It will not be possible to survey the whole sky in all filters and generate templates by the end of the commissioning period. 
+A strategy for template generation in the early phases of the survey, which will require balancing a tradeoff between various factors such as smaller area with multiple filters vs a single filter over a large area, must be devised. 
+Different science drivers naturally lead to different prioritization strategies, e.g. milky way science would prefer from templates that cover the galactic plane, time domain science would prefer templates in multiple bands rather than a single band for a larger area. 
+Supernova, transient and variable science strongly advocate for templates for all bands in the Deep Drilling Fields to be prioritized. 
+Rubin Operations will work closely with the science community to develop a science-driven approach to acquiring on-sky observations and template generation in commissioning and the early phases of the survey that will benefit the maximum number of science cases. 
```

Comment:
We don't mention DP1 and ComCam. 
maybe add here apointer to 6.3 for ComCam? No live alerts

Reply:
Not in the science section, no. I'm not sure what you want me to add? 
Following a discussion with Keith, I removed commissioning as a) template genreation is not an explicit goal of commissioning and b) comissioning have already sought input from the community on acquiring on-sky observations so this might be confusing.   We will work  with the community to develop the template generation strategy for y1,

Reply:
Will follow up offline